mmv1/products/blockchainnodeengine/BlockchainNodes.yaml

```
@@ -0,0 +1,211 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BlockchainNodes'
+base_url: projects/{{project}}/locations/{{location}}/blockchainNodes
+self_link: 'projects/{{project}}/locations/{{location}}/blockchainNodes/{{blockchain_node_id}}'
+import_format: ['projects/{{project}}/locations/{{location}}/blockchainNodes/{{blockchain_node_id}}']
+create_url: projects/{{project}}/locations/{{location}}/blockchainNodes?blockchain_node_id={{blockchain_node_id}}
+description: |
+  A representation of a blockchain node.
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 45
+  delete_minutes: 35
+
+async: !ruby/object:Api::OpAsync
```

```suggestion
autogen_async: true
async: !ruby/object:Api::OpAsync
```

Have a go adding this line and deleting `mmv1/third_party/terraform/services/blockchainnodeengine/blockchainnodeengine_operation.go.erb` - do you definitely need to have that file present in this PR? You can use `make vet` on the generated provider codebase to check that the new, generated operation code works with the other code before pushing to the PR.

---

looks to be working with `autogen_sync`, this code was worked on prior to having a better understanding of async calls